pdp8.py

```diff
@@ -540,10 +541,10 @@ def _CIL(self):
             self.microistruzioni += "E-AC <- AC-E , "
             self.microistruzioni += "F<- 0 \n"
             self.microistruzioni += "----- \n"
-    
+
     def _SPA(self):
         """
-        Salta l'istruzione successiva se il contenuto di AC > 0
+        Salta l'istruzione successiva se il contenuto di AC è positivo o pari a 0

```

Comment:
Questo PDP8 è una versione rivisitata per scopi didattici, non è completamente equivalente a quello reale e non ha il set OPR 2. Quindi non posso accettare questo cambiamento perché il fine di questa istruzione è proprio di vedere se il contenuto è positivo.

L'OPR originale del PDP8 aveva un comportamento totalmente diverso, controllava solo il segno del complemento a  2. Come ho detto nel commento precedente questa versione è per uso didattico e non è uno ad uno con l'originale.

Reply:
Questo è di fatti ciò che ho fatto io, ovvero controllare (come da manuale) unicamente il bit del segno.
Ad ogni modo a me in realtà piace anche la tua versione adattata a scopo didattico, ma comunque si può notare che si potrebbe comunque ottenere il tuo effetto anche con questa istruzione! Ci sono diversi modi credo, uno che mi viene in mente ora è usare in concatenazione prima SPA e poi nel caso salti l'istruzione, SZA per accertarsi che non sia 0. Quindi alla fine bene o male un risultato si ottiene comunque, ma resta a te la scelta finale chiaramente

Reply:
Bene, magari più avanti si cambiano anche le altre...